### PR TITLE
[SONARMSBRU-58] Document requirement to install TFS2013 OM on TFS2015 build agent

### PR DESCRIPTION
**Pull request by duncanp-sonar**
Amended the documentation to refer to both TFS2013 and TFS2015.
Added short section on configuring an analysis build in the new build system using command steps.

### CODE REVIEW

File: doc/analyze-from-tfs.md [hunk 1]
```diff
@@ -1,11 +1,43 @@
...
+
+#### Analyzing code stored in Visual Studio Online ("VSO") requires a TFS 2013 build agent
+If you are analysing code stored in VSO using a XAML build then at present you must use TFS 2013 build agent.
+This is a known issue that is being tracked here [http://jira.sonarsource.com/browse/SONARMSBRU-73].
+
+
+### Updating an existing XAML build definition
 
 **>> NOTE >> Assumptions**:
 - One of the standard Team Build workflow templates for TFS2013 (GitTemplate.12.xaml or TfvcTemplate.12.xaml) and that the standard Microsoft build targets are used. Users who have customized either the build targets or workflow templates may need to modify the following steps to take account of their customizations.
```
Comment (duncanp-sonar) [inline doc/analyze-from-tfs.md:44]: The names of the templates haven't changed in TFS 2015.

File: doc/analyze-from-tfs.md [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+
 ### Mapping Build Definitions to SonarQube projects
 
 SonarQube uses *Projects* to organize analysis results by logical application, where an application can consist of a number of *modules* (assemblies). It is not currently possible to upload partial analysis results for a SonarQube Project. For example, if SonarQube project *X* consists of assemblies *A*, *B* and *C*, it is not possible to build, analyze and upload data for *A* and *B*, and later to build, analyze and upload data for *C*.
 
 This means that a Build Definition must build and analyze all of the assemblies that are in that SonarQube Project.
 
-### Update the build definition
+
+## Analyzing projects in XAML Builds in TFS 2013 and TFS 2015
```
Comment (bgavrilMS) [inline doc/analyze-from-tfs.md:18]: I would add a line saying that VS 2013 also installs the OM

**Comment by bgavrilMS** [comment]
LGTM

File: doc/analyze-from-tfs.md (hunk 2 above)
Comment (duncanp-sonar) [inline doc/analyze-from-tfs.md:18]: Done.

**Pull request merged**